From: Dmitry Shachnev <mitya57@debian.org>
Date: Sat, 7 Jan 2023 16:46:57 +0400
Subject: Fix AttributeError: module 'importlib' has no attribute 'machinery'

Origin: https://github.com/archlinux/svntogit-packages/blob/packages/pyside2/trunk/fix-build.patch
---
 sources/shiboken2/libshiboken/embed/signature_bootstrap.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/sources/shiboken2/libshiboken/embed/signature_bootstrap.py b/sources/shiboken2/libshiboken/embed/signature_bootstrap.py
index 9028642..c70abc5 100644
--- a/sources/shiboken2/libshiboken/embed/signature_bootstrap.py
+++ b/sources/shiboken2/libshiboken/embed/signature_bootstrap.py
@@ -211,7 +211,7 @@ class EmbeddableZipImporter(object):
         return self if self._mod2path.get(fullname) else None
 
     def load_module(self, fullname):
-        import importlib
+        import importlib.machinery
         import sys
 
         filename = self._mod2path.get(fullname)
